src/mapkit.cs

```diff
@@ -1828,5 +1885,101 @@ interface NSUserActivity_MKMapItem {
 		[Export ("setMapItem:")]
 		void SetMapItem (MKMapItem item);
 	}
+
+[TV (11,0)][NoWatch][iOS (11,0)][Mac (10,13, onlyOn64: true)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface MKClusterAnnotation : MKAnnotation {
+		[NullAllowed, Export ("title")]
+		new string Title { get; set; }
+
+		[NullAllowed, Export ("subtitle")]
+		new string Subtitle { get; set; }
+
+		[Export ("memberAnnotations")]
+		IMKAnnotation[] MemberAnnotations { get; }
+
+		[Export ("initWithMemberAnnotations:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (IMKAnnotation[] memberAnnotations);
+	}
+
+	[NoTV][iOS (11,0)][NoMac][NoWatch]
+	[BaseType (typeof (UIView))]
+	[DisableDefaultCtor]
+	interface MKCompassButton {
+		[Static]
+		[Export ("compassButtonWithMapView:")]
+		MKCompassButton FromMapView ([NullAllowed] MKMapView mapView);
+
+		[NullAllowed, Export ("mapView", ArgumentSemantic.Weak)]
+		MKMapView MapView { get; set; }
+
+		[Export ("compassVisibility", ArgumentSemantic.Assign)]
+		MKFeatureVisibility CompassVisibility { get; set; }
+	}
+
+	[TV (11,0)][NoWatch][iOS (11,0)][NoMac]
+	[BaseType (typeof (MKAnnotationView))]
+	interface MKMarkerAnnotationView {
+
+		[Export ("titleVisibility", ArgumentSemantic.Assign)]
+		MKFeatureVisibility TitleVisibility { get; set; }
+
+		[Export ("subtitleVisibility", ArgumentSemantic.Assign)]
+		MKFeatureVisibility SubtitleVisibility { get; set; }
+
+		[Appearance]
+		[NullAllowed, Export ("markerTintColor", ArgumentSemantic.Copy)]
+		UIColor MarkerTintColor { get; set; }
+
+		[Appearance]
+		[NullAllowed, Export ("glyphTintColor", ArgumentSemantic.Copy)]
+		UIColor GlyphTintColor { get; set; }
+
+		[Appearance]
+		[NullAllowed, Export ("glyphText")]
+		string GlyphText { get; set; }
+
+		[Appearance]
+		[NullAllowed, Export ("glyphImage", ArgumentSemantic.Copy)]
+		UIImage GlyphImage { get; set; }
+
+		[Appearance]
+		[NullAllowed, Export ("selectedGlyphImage", ArgumentSemantic.Copy)]
+		UIImage SelectedGlyphImage { get; set; }
+
+		[Export ("animatesWhenAdded")]
+		bool AnimatesWhenAdded { get; set; }
+	}
+
+	[TV (11,0)][NoWatch][iOS (11,0)][NoMac]
+	[BaseType (typeof (UIView))]
+	interface MKScaleView {
+
+		[Static]
+		[Export ("scaleViewWithMapView:")]
+		MKScaleView FromMapView ([NullAllowed] MKMapView mapView);
+
+		[NullAllowed, Export ("mapView", ArgumentSemantic.Weak)]
+		MKMapView MapView { get; set; }
+
+		[Export ("scaleVisibility", ArgumentSemantic.Assign)]
+		MKFeatureVisibility ScaleVisibility { get; set; }
+
+		[Export ("legendAlignment", ArgumentSemantic.Assign)]
+		MKScaleViewAlignment LegendAlignment { get; set; }
+	}
+
+	[NoTV][iOS (11,0)][NoWatch][NoMac]
+	[BaseType (typeof (UIView))]
```

Comment:
Based on the API to create this UI item I do not think you're allowed to set the frame of that UI element. If I'm wrong this it's easy to add later. However good call on `[DisableDefaultCtor]`.